Sources/TuistGenerator/Mappers/AutogeneratedProjectSchemeWorkspaceMapper.swift

```diff
@@ -5,7 +5,13 @@ import TuistGraph
 /// Mapper that generates a new scheme `ProjectName-Project` that includes all targets
 /// From a given workspace
 public final class AutogeneratedProjectSchemeWorkspaceMapper: WorkspaceMapping {
-    public init() {}
+    private let enableCodeCoverage: Bool
+
+    // MARK: - Init
+
+    public init(config: Config) {
+        self.enableCodeCoverage = config.generationOptions.contains(.enableCodeCoverage)
+    }
```

Comment:
minor: perhaps here we could only pass the `enableCodeCoverage` boolean or the `generationOptions` seeing that's the only requirement? 

_(my slightly odd reasoning 😅 .. `Config` has many other concerns that are for different features and modules - e.g. Plugins, Cloud, etc... )_

Reply:
Sounds good to me, I initially though that passing Config would make sense as there are other "project" setting, yet again, a part from what contains, it would still be better that AutogeneratedProjectSchemeWorkspaceMapper only has knowledge of what it needs :) Will change.